api/app/Traits/Generator/GeneratesUserDoc.php

```diff
@@ -0,0 +1,418 @@
+        $this->addLabelText($section, $this->localizeHeading('work_location'), $this->localizeEnumArray($user->location_preferences, WorkRegion::class));
+        $this->addLabelText($section, $this->localizeHeading('location_exemptions'), $user->location_exemptions ?? '');
+    }
+
+    /**
+     * Generate information about a users work preferences
+     *
+     * @param  Section  $section  The section to add info to
+     * @param  User  $user  The user being generated
+     */
+    protected function workPreferences(Section $section, User $user, $headingRank = 4)
+    {
+        $section->addTitle($this->localizeHeading('work_preferences'), $headingRank);
+        foreach ($user?->position_duration ?? [] as $duration) {
+            $section->addListItem($this->localizeEnum($duration, PositionDuration::class));
+        }
+
+        $preferences = $user->getOperationalRequirements();
+        if (count($preferences['accepted']) > 0) {
+            $section->addText($this->localizeHeading('accepted_operational_requirements'), $this->strong);
+
+            foreach ($preferences['accepted'] as $preference) {
+                $section->addListItem($this->localizeEnum($preference, OperationalRequirement::class, 'long'));
+            }
+        }
+
+        if (count($preferences['not_accepted']) > 0) {
+            $section->addText($this->localizeHeading('rejected_operational_requirements'), $this->strong);
+
+            foreach ($preferences['not_accepted'] as $preference) {
+                $section->addListItem($this->localizeEnum($preference, OperationalRequirement::class, 'long'));
+            }
+        }
+    }
+
+    /**
+     * Generate Diversity, equity and inclusion info for  auser
+     *
+     * @param  Section  $section  The section to add info to
+     * @param  User  $user  The user being generated
+     */
+    protected function dei(Section $section, User $user, $headingRank = 4)
+    {
+        $section->addTitle($this->localizeHeading('dei'), $headingRank);
+
+        if ($user->indigenous_communities) {
+            foreach ($user->indigenous_communities as $community) {
+                $section->addListItem($this->localizeEnum($community, IndigenousCommunity::class, 'long'));
+            }
+        }
+        if ($user->is_woman) {
+            $section->addListItem($this->localizeHeading('woman'));
+        }
+        if ($user->is_visible_minority) {
+            $section->addListItem($this->localizeHeading('visible_minority'));
+        }
+        if ($user->has_disability) {
+            $section->addListItem($this->localizeHeading('disability'));
+        }
+    }
+
+    /**
+     * Generate a users experiences
+     *
+     * @param  Section  $section  The section to add info to
+     * @param  User  $user  The user being generated
+     */
+    protected function experiences(Section $section, User $user, $headingRank = 3)
+    {
+
+        if ($user->experiences->count() > 0) {
```

Comment:
Formatting of experiences is a little hard to parse, hard to tell where one begins and ends

![image](https://github.com/user-attachments/assets/52a58491-5043-4a51-b636-2d42dc7fb330)

Reply:
Oof, yeah that is bad. I tightened it up but it is still kinda hard to read :cry: I tihnk it is better at least :woman_shrugging: 

[fix headings, experience spacing](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11151/commits/9ea766020d8987729f9b84b5050c074d5a5aa8db)

Reply:
lol you actually uncovered a bug :sweat_smile: Might look a bit better after [fix h5 style](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11151/commits/de218ae9c47cb787a246e2611c0e959b04a8cee2)

Reply:
Well that looks much nicer